### PR TITLE
Stopped CircleCi Build for master branch

### PR DESCRIPTION
**Pull request by gdsoumya**
Signed-off-by: gdsoumya <gdsoumya@gmail.com>

**What this PR does / why we need it**:

This PR stops the Litmus Portal CI builds on master branch

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Checklist**

* [ ] Does this PR have a corresponding GitHub issue?
* [ ] Have you included relevant README for the chaoslib/experiment with details?
* [ ] Have you added debug messages where necessary? 
* [ ] Have you added task comments where necessary? 
* [ ] Have you tested the changes for possible failure conditions?
* [ ] Have you provided the positive & negative test logs for the litmusbook execution?
* [ ] Does the litmusbook ensure idempotency of cluster state?, i.e., is cluster restored to original state?
* [ ] Have you used non-shell/command modules for Kubernetes tasks?
* [ ] Have you (jinja) templatized custom scripts that is run by the litmusbook, if any? 
* [ ] Have you (jinja) templatized Kubernetes deployment manifests used by the litmusbook, if any?
* [ ] Have you reused/created util functions instead of repeating tasks in the litmusbook?
* [ ] Do the artifacts follow the appropriate directory structure? 
* [ ] Have you isolated storage (eg: OpenEBS) specific implementations, checks? 
* [ ] Have you isolated platform (eg: baremetal kubeadm/openshift/aws/gcloud) specific implementations, checks?  
* [ ] Are the ansible facts well defined? Is the scope explicitly set for playbook & included utils?
* [ ] Have you ensured minimum/careful usage of shell utilities (awk, grep, sed, cut, xargs etc.,)?
* [ ] Can the limtusbook be executed both from within & outside a container (configurable paths, no hardcode)?
* [ ] Can you suggest the minimal resource requirements for the litmusbook execution?
* [ ] Does the litmusbook job artifact carry comments/default options/range for the ENV tunables?
* [ ] Has the litmusbooks been linted? 

**Special notes for your reviewer**:

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+            ignore:   
+                - master
+        machine: true
+        steps:
+            - checkout
+
+            # build the application image
+            - run: |
+                  echo "$DPASS" | docker login --username $DUNAME --password-stdin
+                  cd litmus-portal && make build push
```
Comment (rajdas98) [inline .circleci/config.yml:14]: Remove This part

Comment (gdsoumya) [reply .circleci/config.yml]: It shouldn't be a problem as the the workflow will not be executed in the master, also we need some run command or ci throws an error. I have added a simple echo as a run command.

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+version: 2
+jobs:
+    build:
+        branches:  
+            ignore:   
+                - master
+        machine: true
```
Comment (ksatchit) [inline .circleci/config.yml:7]: Is this needed !

Comment (gdsoumya) [reply .circleci/config.yml:7]: Yes @ksatchit I removed each instruction and checked, every time it threw an error. I believe the current version is the minimal config that we can go for.

**Pull request merged**